kingpin/actors/aws/cloudformation.py

```diff
@@ -40,6 +43,11 @@
 
 __author__ = 'Matt Wise <matt@nextdoor.com>'
 
+class DateTimeEncoder(JSONEncoder):
+    #Override the default method
+    def default(self, obj):
+        if isinstance(obj, (datetime.date, datetime.datetime)):
+            return obj.isoformat()
```

Comment:
isn't this going to turn something like `2010-12-10` into `2010-12-10T00:00:00`?

I think `_parse_policy_json` would need to not parse `2010-12-10` as a datetime in the first place but that seems like a potentially much more intrusive change.

Reply:
Let me actually remove the `datetime.datetime` support. These end up as `datetime.date`.

```
import datetime
datetime.date.fromisoformat('2010-12-10').isoformat()
'2010-12-10'
```

But, I am definitely a little worried about this. In the Boneman change, would a change in a policy version show up in the diff?

Reply:
So AWS hasn't changed the "version"  since 2010.. so just FWIW, it's not likely that we will be changing that string any time soon.